docs/release_notes.rst

```diff
@@ -0,0 +1,5 @@
+.. release-notes:: Release Notes
+    :collapse-pre-releases:
+    :ignore-notes:
+        fix-to_networkx-a78f0f669cc9638c.yaml
+    :reporoot: ../
```

Comment:
Is this needed? The `sphinx-build` command is run from the parent directory

Reply:
This is relative to the `docs/` directory, the repo root is one outside of that.